bridge/bridge/models/BridgeConfiguration.py

```diff
@@ -12,6 +13,14 @@
 ])
 
 
+class StrategyMode(Enum):
+	"""Global strategy mode."""
+
+	STAKE = 1
+	WRAP = 2
+	SWAP = 4
```

Comment:
Why not 3?

Reply:
left this so it would be bitmask compliant (but probably not needed)